metrics/micrometer/src/main/java/com/datastax/oss/driver/internal/metrics/micrometer/MicrometerNodeMetricUpdater.java

```diff
@@ -97,7 +97,7 @@ protected void cancelMetricsExpirationTimeout() {
   protected Timer.Builder configureTimer(Timer.Builder builder, NodeMetric metric, MetricId id) {
     DriverExecutionProfile profile = context.getConfig().getDefaultProfile();
     if (metric == DefaultNodeMetric.CQL_MESSAGES) {
-      return builder
+      builder
           .publishPercentileHistogram()
```

Comment:
Regarding second problem. we can always whitelist what kind metrics we allow. That is what we are doing in Walmart.  so we already have solution for second. So also this is configurable and adding new 3 metrics series.

Reply:
also am ok to remove publishPercentileHistogram if percentiles are define. let me know if that is the path we want to go.

Reply:
Yes. because too many metrics, none of the walmart teams are enabled the metrics.
As i understand correctly, you would like two config one for "publish local percentiles" and another one for ""publish aggregable histogram"
i am ok to that.. correct me if I am wrong.

Reply:
Another general clarification i would like to see that is this going to be merged before handing to over apache foundation?

Reply:
After some more thought and discussion on our side I think we've landed on one new configuration option which switches aggregable histogram generation on/off for all metric flavors [default=on], in addition to the per-metric publishPercentile settings you have already pulled together in this PR. If you could add that to this PR and tackle some of the other newly added feedback around doc strings then I think we'll be good to go here :)

Reply:
Thank you. added addtional configuration option for aggregable histogram generation

Reply:
Note could not add junit cases for this new configuration as micrometer SimpleMeterRegistry does not generate _bucket metrics . I have changes ready to add junit test cases but it requires PrometheusMeterRegistry.

Reply:
i have this junit test case id different branch. It needs micrometer-registry-prometheus dependecy. I can pull junit cases in this PR if thats ok
https://github.com/nparaddi-walmart/java-driver/blob/f5a9b39ff37831a61b8dcd0f4ba5e871fa2ccf8c/metrics/micrometer/src/test/java/com/datastax/oss/driver/internal/metrics/micrometer/MicrometerNodeMetricUpdaterTest.java#L294